<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QMenuBar" name="menubar">
      <widget class="QMenu" name="fileMenu">
       <property name="title">
        <string>File</string>
       </property>
       <addaction name="actionLoad_Model"/>
       <addaction name="actionLoad_Texture"/>
       <addaction name="actionChose_Color"/>
       <addaction name="actionSave_Model"/>
       <addaction name="actionTriangulation"/>
       <addaction name="actionRotate_Scale_Translate"/>
       <addaction name="actiondsad"/>
      </widget>
      <widget class="QMenu" name="moveMenu">
       <property name="title">
        <string>Move</string>
       </property>
       <addaction name="actionUp"/>
       <addaction name="actionDown"/>
       <addaction name="actionLeft"/>
       <addaction name="actionRight"/>
       <addaction name="actionBack"/>
       <addaction name="actionForward"/>
      </widget>
      <addaction name="fileMenu"/>
      <addaction name="moveMenu"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,10,1">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="2,12,1,2,12">
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;Models list&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>91</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Use</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>0</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;Camera list&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Delet mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Normal mode</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_show_mesh">
              <property name="text">
               <string>Show mesh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_show_texture">
              <property name="text">
               <string>Show texture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_show_illumination">
              <property name="text">
               <string>Show illlumination</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>add Texture</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CustomView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="scale">
          <property name="text">
           <string>Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="on_rotate_clicked">
          <property name="text">
           <string>Rotate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="translate">
          <property name="text">
           <string>Translate</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStatusBar" name="statusbar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
   <action name="actionTriangulation">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Triangulation</string>
    </property>
    <property name="toolTip">
     <string>triangulation</string>
    </property>
   </action>
   <action name="actionLoad_Model">
    <property name="text">
     <string>Load Model</string>
    </property>
   </action>
   <action name="actionLoad_Texture">
    <property name="text">
     <string>Load Texture</string>
    </property>
   </action>
   <action name="actionChose_Color">
    <property name="text">
     <string>Chose Color</string>
    </property>
   </action>
   <action name="actionSave_Model">
    <property name="text">
     <string>Save Model</string>
    </property>
   </action>
   <action name="actionUp">
    <property name="text">
     <string>Up</string>
    </property>
    <property name="shortcut">
     <string>Up</string>
    </property>
   </action>
   <action name="actionDown">
    <property name="text">
     <string>Down</string>
    </property>
    <property name="shortcut">
     <string>Down</string>
    </property>
   </action>
   <action name="actionLeft">
    <property name="text">
     <string>Left</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </action>
   <action name="actionRight">
    <property name="text">
     <string>Right</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </action>
   <action name="actionBack">
    <property name="text">
     <string>Back</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </action>
   <action name="actionForward">
    <property name="text">
     <string>Forward</string>
    </property>
    <property name="shortcut">
     <string>W</string>
    </property>
   </action>
   <action name="actionRotate_Scale_Translate">
    <property name="text">
     <string>Rotate_Scale_Translate</string>
    </property>
   </action>
   <action name="actiondsad">
    <property name="text">
     <string>dsad</string>
    </property>
   </action>
  </widget>
  <action name="actionRotateMesh">
   <property name="text">
    <string>TranslateMesh</string>
   </property>
  </action>
  <action name="actionTranslateMesh">
   <property name="text">
    <string>TranslateMesh</string>
   </property>
  </action>
  <action name="actionTranslateMesh_2">
   <property name="text">
    <string>TranslateMesh</string>
   </property>
  </action>
  <action name="actionTranslate_Mesh">
   <property name="text">
    <string>Translate_Mesh</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
